.github/workflows/build_lint.yml

```diff
@@ -38,30 +35,27 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v3
+      uses: actions/setup-python@v4
       with:
         python-version: ${{ matrix.python-version }}
     - uses: actions/cache@v3
       with:
         path: ~/.cache/pip
         key: ${{ runner.os }}-pip-${{ matrix.python-version }}-${{ hashFiles('Pipfile.lock') }}
     - name: Install dependencies
-      run: |
-        sudo apt-get install libspatialindex-dev
```

Comment:
Why is `libspatialindex-dev` no longer needed?

Reply:
Didn't really know or remember why it was here. I guess it was required by `geopandas`? [geopandas doc](https://geopandas.org/en/stable/getting_started/install.html#dependencies) says

> [rtree](https://github.com/Toblerity/rtree) (optional; spatial index to improve performance and required for overlay operations; interface to [libspatialindex](https://github.com/libspatialindex/libspatialindex))

and [rtree doc](https://github.com/Toblerity/rtree) says

> Wheels are available for most major platforms, and rtree with bundled libspatialindex can be installed via pip:
> 
> `pip install rtree`

`rtree` is one of our dependencies now:

https://github.com/wang-boyu/mesa-geo/blob/9f28f7c8a83ced81b44b19083ab5a300804acc76/setup.cfg#L32-L36

So I think `libspatialindex` is no longer required to be installed manually.